validator/src/main.rs

```diff
@@ -2225,6 +2279,23 @@ pub fn main() {
                 });
             return;
         }
+        ("set-block-engine-config", Some(subcommand_matches)) => {
+            let block_engine_url = value_t_or_exit!(subcommand_matches, "block_engine_url", String);
+            let trust_packets = subcommand_matches.is_present("trust_block_engine_packets");
+            let admin_client = admin_rpc_service::connect(&ledger_path);
+            admin_rpc_service::runtime()
+                .block_on(async move {
+                    admin_client
+                        .await?
+                        .set_block_engine_config(block_engine_url, trust_packets)
+                        .await
+                })
+                .unwrap_or_else(|err| {
+                    println!("set block engine config failed: {}", err);
+                    exit(1);
+                });
```

Comment:
```suggestion
                .expect("set block engine config failed");
```
expect() will include the error, eg

`    let a = "5a".parse::<i64>().expect("hi");`
returns
```
thread 'main' panicked at 'hi: ParseIntError { kind: InvalidDigit }', src/benchmark_datastructure.rs:63:33
```

Reply:
tru, but gonna stick with pattern already established in other commands